easybuild/easyblocks/e/esmf.py

```diff
@@ -60,6 +60,10 @@ def configure_step(self):
             compiler = comp_family.lower()
         env.setvar('ESMF_COMPILER', compiler)
 
+        if self.toolchain.options.get('optarch'):
+            env.setvar('ESMF_F90COMPILEOPTS', os.getenv('F90FLAGS'))
+            env.setvar('ESMF_CXXCOMPILEOPTS', os.getenv('CXXFLAGS'))
```

Comment:
@lexming why only set these envars if optarch is set?

Reply:
to still allow for `optarch = False` and not tune the build in any way

Reply:
ok but even if `optarch = False`, the EB flags would be useful/preferred since, as you show, a lot more unit tests pass if you set them? or are all failing tests optarch related?

Reply:
I tested this option and the results with `optarch = False` are equally bad with or without the other flags from EB. So it seems that the issue really boils down to just the `optarch` and we can add our flags every time. Fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/2325/commits/a7282645d463fb98c59d4a85620cb7fc825ebdc8